packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/Actions.tsx

```diff
@@ -138,7 +138,6 @@ const BulkActionAction = (action: Action) => {
           break;
         case 'dialog':
         case 'modal': {
-          if (action.actionType === 'delete') trackUsage('willBulkDeleteEntries');
```

Comment:
I changed `type` to `actionType` because I needed to do this 😥 and I polluted it. Thanks for the changes!!

Reply:
You could add this to the action call of `delete` before the API request is sent?